### DISCUSSION TITLE
Getting MacOS running for testing

### DISCUSSION THREAD

**Discussion by PQCraft** [PQCraft/CLIBASIC#5] [Q&A] [answered]
I may or may have not attempted to install MacOS on a VM and had it hang after running `System/Library/CoreServices/boot.efi` from the EFI shell. In the case that someone has an answer to this theoretical question, pls comment it.

**Comment by DamieFC** [comment]
My answer: why are you running a system file and CLIBASIC at the same time lol?

**Reply by PQCraft** [reply]
Ah no, that hypothetically was to try to boot MacOS. I hypothetically have it installed thanks to https://github.com/kholia/OSX-KVM.

**Reply by PQCraft** [reply]
I'm going to close this now lol

**Comment by PQCraft** [comment, marked as answer]
https://github.com/kholia/OSX-KVM
